src/connector/passive.js

```diff
@@ -79,6 +79,10 @@ class Passive extends Connector {
           }
         });
       });
+    })
+    .catch((error) => {
+      this.log.trace(error.message);
+      return this.setupServer();
```

Comment:
Is this to try again if it fails? What if it perpetually fails forever?

Reply:
What would you do here then? Just remove this line totally?

Reply:
Yeah I would let it fail. Disconnect and/or respond with an error code to the client.